alcs-api/src/application/application-decision-maker/application-decision-maker.service.ts

```diff
@@ -0,0 +1,24 @@
+import { Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { ApplicationDecisionMaker } from './application-decision-maker.entity';
+
+@Injectable()
+export class ApplicationDecisionMakerService {
+  constructor(
+    @InjectRepository(ApplicationDecisionMaker)
+    private readonly applicationTypeRepository: Repository<ApplicationDecisionMaker>,
```

Comment:
I thought we getting read of this 'readonly'

Reply:
Woops, removed.